src/CloudModelGenerator/Program.cs

```diff
@@ -20,9 +28,19 @@ static int Main(string[] args)
             var includeTypeProvider = app.Option("-t|--withtypeprovider", "Indicates whether the CustomTypeProvider class should be generated.", CommandOptionType.NoValue);
             var structuredModel = app.Option("-s|--structuredmodel", "Indicates whether the classes should be generated with types that represent structured data model.", CommandOptionType.NoValue);
 
+
             app.OnExecute(() =>
             {
-                if (!projectIdOption.HasValue())
+                // Check if default values are set
+                var passedSetProjectId = configuration["defaultFlags:0:projectId"] ?? projectIdOption.Value();
+                var passedSetNamespace = configuration["defaultFlags:0:namespace"] ?? namespaceOption.Value();
+                var passedSetOutputDir = configuration["defaultFlags:0:outputdir"] ?? outputDirOption.Value();
+                var passedSetFileNameSuffix = configuration["defaultFlags:0:filenameSuffix"] ?? fileNameSuffixOption.Value();
+                var passedSetIncludeTypeProvider = configuration["defaultFlags:0:withTypeProvider"] ?? includeTypeProvider.Value();
```

Comment:
maybe it would make sense to parse the booleans right here instead of doing it later